lib/methods/export.js

```diff
@@ -115,56 +151,63 @@ module.exports = self => {
         .toArray();
     },
 
-    getRelatedDocsIds(manager, doc, relatedTypes) {
-      // Use `doc.type` for pages to get the actual schema of the corresponding page type.
-      const schema = manager.schema || self.apos.modules[doc.type].schema;
+    getRelatedIdsBySchema({
+      doc, schema, type = 'relationship', relatedTypes, recursion = 0
+    }) {
+      return schema.flatMap(field => {
+        const fieldValue = doc[field.name];
+        const shouldRecurse = recursion < MAX_RECURSION;
+
+        if (
+          !fieldValue ||
+          (relatedTypes && field.withType && !relatedTypes.includes(field.withType))
+          /* ( */
+          /*   type === 'relationship' && */
+          /*   field.withType && */
+          /*   self.apos.modules[field.withType].options.relatedDocument === false */
+          /* ) */
```

Comment:
add a `TODO: handle 'export: false' option`

Reply:
Added (but we should choose another option name)

Reply:
I am handling this in `fetchActualDocs`, actually:

![image](https://github.com/apostrophecms/import-export/assets/8301962/46474084-de52-4d7d-b455-1c1d90d413ab)

Reply:
We should use another option name since this one is already used by `piece-type-exporter` and could occur unclear conflicts.
Also I think it would be more efficient to perform this check before the DB request if we can.

Reply:
ok I'll handle that in https://github.com/apostrophecms/import-export/pull/13